### DISCUSSION TITLE
Number of covered objects in one game

### DISCUSSION THREAD

**Discussion by Chance4us** [diasurgical/DevilutionX#1981] [Ideas]
Since most of covered objects appear in the caves and in the hive the idea arose to simply count them.
What do you think about to add an outline or an another kind of marking to these objects?

The answer for the number is attached. 
[png.zip](https://github.com/diasurgical/devilutionX/files/6487295/png.zip)

**Comment by qndel** [comment]
we could always go with the solution I used in crucible - when you kill all monsters on the level, objects and items are highlighted + shown on the automap

**Reply by Chance4us** [reply]
Yes,
 >> when you kill all monsters on the level, objects and items are highlighted + shown on the automap

would be fine.

**Comment by AJenbo** [comment]
Are the attached screen shots just 25 images of things behind walls? Not much to see, i think one image would have been fine, and even better if it was inserted directly in the text.

**Reply by Chance4us** [reply]
Yes, each picture contains a "hidden" object behind walls.

**Comment by AJenbo** [comment]
Counting what isn't visible to humans isn't really trivial, a more realistic option would be to count unopened items. But I think this more of a mod thing.

**Reply by Chance4us** [reply]
Yes, and I think the idea of @qndel would be fine as an option in the ini file, so that the unopened chests or barrels or
potions/ items are not overlooked.

**Comment by julealgon** [comment]
I feel like this is an immersion killer and would really not want it to be part of the game at all.

There was already discussion around adding proper transparency to caves, and that would be a much more sensible thing to do IMHO.
